acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/models/Publication.java

```diff
@@ -63,20 +63,8 @@ public Granularity getGranularity() {
         return granularity;
     }
 
-    public Instant getNominalDate() {
-        return nominalDate;
-    }
-
-    public String getNominalDateFormatted() {
-        final int weeksUntilCutOff = 8;
-        final int daysInAWeek = 7;
-        final int daysUntilCutOff = weeksUntilCutOff * daysInAWeek;
-
-        String pattern = nominalDate.isAfter(Instant.now().plus(daysUntilCutOff, ChronoUnit.DAYS))
-            ? "MMM yyyy"
-            : "d MMM yyyy";
-
-        return new SimpleDateFormat(pattern).format(new Date(nominalDate.toEpochMilli()));
+    public NominalPublicationDate getNominalPublicationDateDate() {
```

Comment:
"DateDate"?

Reply:
I'm only responsible for mis-naming the field itself, it's the IDE that generated the getter, blindly copying my mistake :-)

Will fix.

Reply:
:joy: 👍

Reply:
Done.